doc/sphinx/addendum/miscellaneous-extensions.rst

```diff
@@ -3,15 +3,15 @@
 .. _miscellaneousextensions:
 
 Miscellaneous extensions
-=======================
+========================
 
+.. what is this about?
 .. contents::
    :local:
    :depth: 1
-----
```

Comment:
This was emitting an "unexpected unindent warning". However, I'd like to understand what this block is supposed to do.

Reply:
This block should have been removed.